HOWTO.dual.gnutls

```diff
@@ -0,0 +1,98 @@
+############################################################################
+#
+# GnuTLS dual versions
+#
+# Works with CentOS 7
+#
+############################################################################
+#
+# Add in alternative GnuTLS support into /usr/local, and put all the
+# libraries into /usr/local/lib.
+#
+# Need the latest of the following packages
+#
+# gmp    (used 6.2.0)  https://gmplib.org/#DOWNLOAD
+# nettle (used 3.6)    https://ftp.gnu.org/gnu/nettle/
+# gnutls (used 3.6.13) https://www.gnutls.org/download.html
+#
+
+GMP_VER=6.2.0
+NETTLE_VER=3.6
+GNUTLS_VER=3.6.13
+
+#
+# gmp
+#
+tar xovf gmp-${GMP_VER}.tar.lz
```

Comment:
At least my vanilla apt-installed GNU tar 1.30 does not come with lzip support, so maybe it would be wise to fall back to the `.xz` compressed archive to avoid unnecessary complexity.

```shell
$ tar xovf gmp-6.2.0.tar.lz 
tar (child): lzip: Cannot exec: No such file or directory
tar (child): Error is not recoverable: exiting now
tar: Child returned status 2
tar: Error is not recoverable: exiting now
```

Reply:
Updated to .xz